### ISSUE TITLE
MWC - Finalize consolidated findings paper

### ISSUE THREAD

**Issue by thisandagain** [MozillaFoundation/research#10]
We need to do a design and final copy editing pass on the consolidated findings paper for MWC.

Dropbox:
`Research/MWC Synthesis/MWC_1602.indd`

Latest PDF:
https://www.dropbox.com/s/5tl151zmyjsd955/MWC_1602.pdf?dl=0

**Comment by LauraReynal** [comment]
Needs the Web Journey from #2 and probably the threshold if possible ? 
Threshold will depend on Ben and GSMA

**Comment by LauraReynal** [comment]
Turns out also needs work on content following latest call - Will dive on it asap.

**Comment by sabrinang** [comment]
I have been working on this and can you give me some feedback?
https://www.dropbox.com/s/9y2iza6yq8pkgk4/MWC_1902.pdf?dl=0
Dropbox Folder: Research/MWC Synthesis/MWC_1902.pdf

**Comment by LauraReynal** [comment]
Hey @sabrinang thanks for the design pass, it looks much better now ! 
A couple of things: 
* page 14 and 15 : The quote got cut on page 15, (Rural bangladesh) Here is the full one : --> 
“Do you go on the internet?
- No, I don’t have my Facebook account yet.
- Sorry I did not mean Facebook. Have you ever been on a Website?
- Not yet, I will make my facebook account next week.”

* page 15: I find it a little weird that the sentence "we found no other use case ...." comes before the quote, which is supposed to be the "use case" mentioned. 

* page 25; Could we add the Mozilla logo next to the GSMA one?

thanks very much !

**Comment by sabrinang** [comment]
updated: https://www.dropbox.com/s/e30f83qifkpojno/MWC_2002.pdf?dl=0
Dropbox folder: Research/MWC Synthesis/MWC_2002.pdf

I also noticed on page 14, it was had quotes that did not have a author/location after it so I added (dhaka, bangladesh) placeholder:
![mwc-feedback](https://cloud.githubusercontent.com/assets/2492510/6309476/ae7bb72c-b902-11e4-909c-8c5f7efe09f7.jpg)
Let me know what should be there or if it is not needed.

**Comment by LauraReynal** [comment]
Add some analysis and facts from the MIT paper into the synthesis 
Add a participant scope graph 
Add a map 
Do a general content pass on the "seriousness"

**Comment by benrito** [comment]
When will we do next review pass? Has @davidascher had a look?

**Comment by LauraReynal** [comment]
@benrito trying to do the pass for tonight. 
Will Let you know !

**Comment by benrito** [comment]
What are the competing priorities?

Given timeline and your PTO Friday, let’s agree on a final review schedule with @davidascher and @benrito kicking in?

Thanks!

**Comment by LauraReynal** [comment]
Agreed ! Final review schedule proposed 

- Wednesday (today) => Laura makes some changes and sends to team by EOD - night
- Thursday => @benrito @davidascher @cecilepompei do a review and finalize the content 
- Friday => @thisandagain does the final Design pass and hosts the PDF online

Competing priority is => getting India Report ready for publication. 
Thanks Ben

**Comment by thisandagain** [comment]
Tracking link to this report from the MOI site here: https://github.com/mozilla/webmaker.org/issues/1174#issuecomment-76038291

**Comment by LauraReynal** [comment]
Progress of the day: :bicyclist: 

- **Combined all my numbers from 3 countries. We talked to 308 participants, spent 10 weeks in the field, in 3 countries and 16 cities.**
-  Made 2 pie charts : number of participants per country + percentage of participants living in rural/urban/others

*In the document* :pencil2: 
- Added goal on p.3
- added the three pillar of the open Web = 1 page of text + the graphic (that Andrew will re-design) p.8 and 9 
- Added the research scope with the new numbers and methods p.12, 13, 14, 15 + UCD graph that Andrew is redoing 
- added the research map

NEXT STEP ?
Rework the Key Trends tomorrow. 
->  Addind some titles, a couple of numbers and maybe a few more stories. 

Meanwhile, feel free to leave comments on the PDF @benrito @davidascher @cecilepompei @thisandagain 
https://www.dropbox.com/s/pw4u8kwa9smbbt0/MWC_2502.pdf?dl=0

**Comment by LauraReynal** [comment]
Pasting @cecilepompei comments here to keep track ! 
@cecilepompei you can leave all comments in the issues, its perfectly fine :smile: 
----
Not sure how to post comments on this so sending comments directly to Laura in French.

Overall, my question for everyone is: didnt they want a two pager? In which case no need to write these intro paragraphs all the time and focus on graphes: 1) who did we sample, 2) what were the main trends and 3) recommendations

To me, p22 onwards are too confusing, and the key trends are not clear. Excellent qualitative research, just would like more quant data to know how big these trends weight.
Laura, sending you detailed comments

Rest, I think it is improving nicely so really well done Laura!

**Comment by LauraReynal** [comment]
Progress of the day :horse_racing: 

- Removed some details on participants - grouped them more 

- Reworked the intro = removed some sentences, added a couple of titles for easier screening

- Added an outline so you would understand what to expect in this paper

- Added a legend on the map 

- Added a title and a legend on the Web journey page

- Key Trends = took out the subtitles / added one "finding title" per trend, similarly to the other reports. Makes it easier to scan the doc and remember the most important points. 

- Added pull out boxes for the 3 stories that are a bit outside of the trends. They illustrate the findings, so it was a bit odd to see the lay out very similar. 

- Added the business card story


------ 
I think the work on content is okay now...but the design is a giant mess

**Comment by LauraReynal** [comment]
Okay. Here is the latest PDF https://www.dropbox.com/s/inpo1fxck8kywcv/MWC_2602.pdf?dl=0

One question : 
Do you prefer participants to be showned in numbered list, or in a lovely pie chart ? @benrito @cecilepompei @thisandagain 
![screen shot 2015-02-26 at 16 42 26](https://cloud.githubusercontent.com/assets/8261945/6396598/3de18fe4-bdd7-11e4-96fa-fbadf65cee52.png)

**Comment by LauraReynal** [comment]
As per latest conversation with @cecilepompei and @benrito I am living the doc for you to remix as you want ! Feel free to make changes.  Please save as a new V.2 documents so I could use this 26_02 version for later. Thanks guys

**Comment by benrito** [comment]
Hey all, made edits and streamlined as much as possible. My file is here:

https://dl.dropboxusercontent.com/u/8176339/ben_edit.indd

However, I have an old version of inDesign and I don't have the assets. So someone with the whole package will need to do another pass to make sure the graphics look OK.

@LauraReynal can you take 5 to incorporate and pass off to @thisandagain ?

**Comment by benrito** [comment]
@davidascher offered to do this, but I think that some of the assets might be on @LauraReynal desktop.

(i.e., Laura's package looks incomplete to me, but it could just be my old version:  https://www.dropbox.com/sh/3dwdgsckir853n2/AADCC2otVJDfsGt58fxn-rWga/MWC%20Synthesis?dl=0)

Therefore, I believe @LauraReynal needs to package the file like this: https://helpx.adobe.com/indesign/how-to/indesign-package-files-for-handoff.html

And then David can open up my edit: https://dl.dropboxusercontent.com/u/8176339/ben_edit.indd (which is just the markup)

And then can re-link with the assets in the full package. This is usually automatic but https://helpx.adobe.com/indesign/using/graphics-links.html might be a helpful reference. @thisandagain is pretty experienced here too.

Let me know if that doesn't make sense, but hopefully it's straightforward.

Cheers!
Ben

**Comment by thisandagain** [comment]
Darn. Yup... happy to help package but it certainly looks like @LauraReynal has assets linked to her local drive. @LauraReynal just ping me on Skype and I can walk you through it.

**Comment by thisandagain** [comment]
Ok... I was able to rebuild by re-linking all the assets to prior reports. I have packaged and updated the directory. Please use `MWC_20150226.indd`. Let me know if you run into any other issues.

**Comment by LauraReynal** [comment]
Hi all sorry I am just seeing this.
Thanks @thisandagain for linking all.

Is the file okay now ? All the assets used are normally in the Dropbox research folder. 
Maybe I used a couple of screenshots for the pie chart but they were placeholders until Andrew designs them with our style guide. 

@benrito,  I gave you the whole research folder but it has so many pictures it's likely that it took time to sync hence the file problems

Let me know if there is anything else


Laura de Reynal
Webmaker
